# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/moz.build
# Commit: 5f90ad3d8781
# Full Hash: 5f90ad3d87818a0810f74bde3bfc71d485eea801
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1746104
# File Overlap Count: 1
# Description:
#   Bug 1746104 - part 5-1: Get rid of `WidgetQueryContentEvent::Reply::mHasSelection` r=m_kato
#   
#   It's intended to indicate whether the selection is collapsed or not, but it can
#   be referred by other members, there is no reasonable user and the name makes
#   developers confused.
# ==============================================================================

diff -r dc81d96d0adb -r 5f90ad3d8781 widget/moz.build
--- a/widget/moz.build	Mon Feb 07 22:33:37 2022 +0000
+++ b/widget/moz.build	Mon Feb 07 22:33:37 2022 +0000
@@ -28,6 +28,9 @@
 with Files("*ContentData*"):
     BUG_COMPONENT = ("Core", "DOM: UI Events & Focus Handling")
 
+with Files("*Events.h"):
+    BUG_COMPONENT = ("Core", "DOM: UI Events & Focus Handling")
+
 with Files("*FontRange*"):
     BUG_COMPONENT = ("Core", "Widget: Cocoa")
 